### DISCUSSION TITLE
Who was the first person to win two Nobel Prizes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#555] [Galaxy Trivia] [answered]
**Bonus:** For what accomplishments in what fields?

**Comment by peterbaccaro** [comment, marked as answer]
The first person in history to achieve the feat of receiving a double Nobel was the Polish **Marie Skłodowska Curie**, laureate first in Physics and, later, in Chemistry.

The only two-time winner of a Nobel Prize not shared with anyone has been Linus Pauling. The first award, the 1954 Nobel Prize in Chemistry, recognized his research on the nature of the chemical bond. And eight years later, his militant pacifism during the Cold War, focused above all on combating nuclear weapons, earned him the Nobel Peace Prize (1962).

**Reply by Veercodeprog** [reply]
@peterbaccaro  you're invited to take part in [discussions](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/2) for coding problems.
Challenge is you have to come up with the best time complexity solution.

**Comment by Veercodeprog** [comment]
The first person to win two Nobel Prizes was Marie Curie. She was awarded the Nobel Prize in Physics in 1903, along with her husband Pierre Curie, for their research on radiation, and later received the Nobel Prize in Chemistry in 1911 for her discovery of the elements radium and polonium. Marie Curie remains the only woman to have won two Nobel Prizes, and is considered one of the most important scientists in history for her pioneering work in the field of radioactivity.

**Comment by thuongtruong1009** [comment]
The first person to win two Nobel Prizes was Marie Curie, a physicist and chemist of Polish origin who lived in France. She won her first Nobel Prize in Physics in 1903, along with her husband Pierre Curie and Antoine Henri Becquerel, for their work on radioactivity. In 1911, she received her second Nobel Prize in Chemistry for the discovery of two new elements, polonium and radium. Marie Curie remains the only person to have won Nobel Prizes in two different fields of science.
